lib/Frontend/CompilerInvocation.cpp

```diff
@@ -66,7 +71,11 @@ static void updateRuntimeLibraryPaths(SearchPathOptions &SearchPathOpts,
   llvm::SmallString<128> LibPath(SearchPathOpts.RuntimeResourcePath);
 
   llvm::sys::path::append(LibPath, getPlatformNameForTriple(Triple));
-  SearchPathOpts.RuntimeLibraryPath = LibPath.str();
+  SearchPathOpts.RuntimeLibraryPaths.clear();
+  SearchPathOpts.RuntimeLibraryPaths.push_back(LibPath.str());
+#if defined(__APPLE__) && defined(__MACH__)
```

Comment:
Can you do this in terms of the target triple rather than the host architecture? I know we can only JIT in-process if they're the same, but it's the principle of the thing.

Reply:
Good idea!